content/en/monitors/faq/anomaly-monitor.md

```diff
@@ -0,0 +1,93 @@
+
+### What is the count_default_zero argument?
+
+Previously, Datadog treated count metrics as gauges, and thus interpolated between reported points. This led to some very odd-looking metrics for sparsely reported counts. Anomalies are no longer interpolating between counts, but for legacy monitors, the old behavior is preserved using the `count_default_zero` argument.
+
+### What if I want my count metric treated as a gauge?
+
+Not interpolating between counts makes sense if your count metric is something like errors. However, if you have regularly scheduled jobs that happen every hour, it might make more sense if the metric is not reporting a value of 0.0 between runs. There are two different ways to accomplish this:
+
+1. Set the rollup (found in the advanced options section) to be one hour.
+2. Explicitly set `count_default_zero='false'` using the API.
+
+### How does setting the rollup interval in "Advanced Options" differ from setting it on the query using .rollup()?
+
+If the rollup is set explicitly on the query, the rollup interval option for the anomaly monitor is ignored.
+
+### I don't care if my metric is anomalous if its value is less than X, can I somehow ignore those anomalies?
+
+Create **A**: an anomaly monitor to alert on values above the bounds; and **B**: a separate [metric monitor][2] with a threshold alert to trigger on values greater than X. Then create a [composite monitor][3] on **A && B**.
+
+### Why am I prevented from saving a monitor with the message "alert and alert recovery criteria are such that the monitor can be simultaneously in alert and alert recovery states"?
+
+Setting different windows for the alert and alert recovery periods might lead to an ambiguous state. The alert and alert recovery window sizes should be set such that both cannot be satisfied at the same time. For example, setting an alert threshold at 50% for a 2-hour window (i.e., 1 hour has to be anomalous to trigger the alert) and the [recovery threshold][4] at 50% for a 10-minute window (i.e., 5 minutes have to be non-anomalous to recover) might result in triggering the alert and the alert recovery states simultaneously. If the last 5 minutes are not anomalous but the 1 hour before _was_ anomalous, both the alert and the alert recovery are triggered.
+
+### How does daylight savings affect anomaly detection monitors?
+
+Datadog monitors use UTC time and by default are agnostic to local time zones (e.g. EST, PST, CST). User activity is shifted relative to UTC time because activity typically remains the same for the user's local time. This could be detected as unexpected anomaly. 
```

Comment:
```suggestion
Datadog monitors use UTC time and by default are agnostic to local time zones. User activity is shifted relative to UTC time because activity typically remains the same for the user's local time. This could be detected as an unexpected anomaly. 
```